scripts/types.ts

```diff
@@ -1,10 +1,8 @@
-import type { Technology } from '@site/src/datasets';
-
 export type SDK = {
   /**
    * The name of the technology used for the SDK.
    */
-  name: Technology;
```

Comment:
I changed this to string. An alternative could be to update the technology type to include "Web" and "Node.js" but that would affect the ecosystem page.

Reply:
How widely is `Technology` even used now? Maybe once we convert the "technologies" pages to all just pull the SDK READMEs it won't even be needed.

Reply:
It's used to validate the data that feeds the ecosystem page.